packages/server-wallet/src/models/channel.ts

```diff
@@ -173,6 +177,67 @@ export class Channel extends Model implements RequiredColumns {
     });
   }
 
+  public static directFundingStatus(
```

Comment:
What's the purpose of this as a static function on a model? It doesn't seem to have anything to do with the `Channel` model. Could it be a standalone function?

Reply:
I agree with this point 2f47878de